dspace-api/src/main/java/org/dspace/app/launcher/Readme.md

```diff
@@ -0,0 +1,57 @@
+# Adding new scripts to the dspace CLI 
+
+Scripts can be added to the script launcher using 2 mechanisms
+
+1. Defining a script in [dspace]/config/launcher.xml
+2. Creating a spring bean of type _org.dspace.app.launcher.CommandType_
+
+The advantage of the second mechanism is that a dependency can provide both the script and add it to the launcher without needing to modify the configuration of the receiving party.
+
```

Comment:
I found the following comment in the Jira ticket to be helpful.  This may also be useful to include in the comments.

> Currently to add a command line script launcher.xml needs to be modified. When managing a shared codebase for multiple dspaces this becomes cumbersome because every receiving dspace needs to have the file edited. Being able to add a command line tool by simply adding a jar would be helpful.